README.md

```diff
@@ -57,23 +57,13 @@ npm install -g aws-deploy-script-frontend
 ```
 
 ```
-aws-deploy <enviroment-name> <build-path>
+aws-deploy -env <enviroment-name> -path <build-path>
```

Comment:
@pabloferro Does this work? When i did it yesterday it needed two `-` for whole words and only one for a single char. Like this `--env` or `-e`. I'm pretty sure if you do `-env` it will generate 3 attributes 1 for e, 1 for n and one for v. Placing the enviorment name on the v.